packages/buildbucket-dart/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.0.12
+
+- Adding dependencies for resultdb.
+
+# 1.0.11
+
+- Add resultsDb protos to the lib.
```

Comment:
Shall we include these two changes in one version?

Reply:
Ah you are right. The 1.0.11 is there though.